src/sqlfluff/rules/L010.py

```diff
@@ -191,6 +200,10 @@ def _get_fix(self, segment, fixed_raw):
         """
         return LintFix.replace(segment, [segment.edit(fixed_raw)])
 
+    @staticmethod
+    def _split_comma_separated_string(raw_str: str) -> List[str]:
```

Comment:
This function is pretty generic, and we're repeating it several times in this PR. Should we consider adding it to `BaseRule` instead?

Reply:
Yeah was wondering where best place to put it. That's sounds good to me. Think it's used elsewhere too so can clean those up at same time.